pytorch/training/docker/1.7.1/py3/Dockerfile.cpu

```diff
@@ -83,6 +84,7 @@ RUN curl -L -o ~/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-lat
     typing \
     "pyopenssl>=17.5.0" \
  && conda install -c conda-forge \
+    python=$PYTHON_VERSION \
```

Comment:
This needs to be at line 77, immediately after miniconda installation, like the other dockerfiles.